src/model_config_tests/util.py

```diff
@@ -4,14 +4,19 @@
 import subprocess as sp
 import time
 
+# Time related constants
+SECONDS_IN_MINUTE = 60
+SECONDS_IN_HOUR = SECONDS_IN_MINUTE * 60
+SECONDS_IN_DAY = SECONDS_IN_HOUR * 24
```

Comment:
I kinda feel like these would be `MINUTE_IN_SECONDS`, `HOUR_IN_SECONDS`, `DAY_IN_SECONDS`...Just so it's `<measure>_IN_<unit>` rather than `<unit>_IN_<measure>` - front-loading the important part, I think.

Reply:
That's a good point. I'll change those now